ingest/pkg/deb/settings-template.toml

```diff
@@ -0,0 +1,46 @@
+
+# log settings for the application (RUST_LOG format). Default below
+# 
+# log = "poc_ingest=debug,poc_store=info"
+
+# Mode to run the ingest in. This adjustst the allowed grpc endpoints. "lora" or
+# "mobile". Required
+mode = "lora"
+
+# Token for ingest grpc endpoint bearer authentication. This is required for
```

Comment:
Deprecated api token?

Reply:
As you figured out a few comments later.. mobile data comes from a known source which we add an additional api token to for some added security. Not needed for lora since that's not quite single source data. 

I expect that api token to go away from mobile too as we add a verifier to verify existence of hotspots on chain